crates/locutus-node/src/node/test_utils.rs

```diff
@@ -51,6 +53,13 @@ pub(crate) struct SimNetwork {
     max_connections: usize,
 }
 
+#[derive(Clone)]
+struct NodeSpecs {
+    owned_contracts: Vec<Contract>,
+    non_owned_contracts: u16,
```

Comment:
not sure what is this for, guess is incomplete. But maybe you want to use Vec<ContractKeys> instead

Reply:
yes it is a mistake, it should be Vec<ContractKeys>